test/Microsoft.AspNetCore.Mvc.FunctionalTests/Infrastructure/MvcEncodedTestFixtureOfT.cs

```diff
@@ -16,9 +16,9 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
             base.ConfigureWebHost(builder);
             builder.ConfigureServices(services =>
             {
+                // Routing does not use DI for getting UrlEncoder and always uses UrlEncoder.Default.
```

Comment:
Why change this fixture? Leave this alone in case we use `IHtmlHelper.UrlEncoder` in the future.

Reply:
We could always add it if we want in future.

Reply:
No. The inconsistency will come back to haunt someone expecting tests about encoding to have all test encoders in DI.